docs/aws-sdk-for-php-with-minio.md

```diff
@@ -75,3 +75,60 @@ $presignedRequest = $s3->createPresignedRequest($command, '+10 minutes');
 // Get the actual presigned-url
 $presignedUrl =  (string)  $presignedRequest->getUri();
 ```
+
+## 6. Bucket Policy
```

Comment:
This would look better  --> "Set a Bucket Policy" - to be consistent with above.